_config.yml

```diff
@@ -6,3 +6,8 @@ description: >
 
 # Build settings
 markdown: kramdown
+
+# Google Calendar API settings
+googleCalendar:
+    apiKey: AIzaSyD3Y_GF0P3MSCWZKhkMQ0FrUfKUf07m4U4
```

Comment:
It should be because no authentication takes place with it, so it defaults to readonly. Reading through the docs, all modifications require an elevated oauth scope. The only reason these events are visible in the API is because the calendar is public. I would assume that unless the user has been invited to the calendar with write permissions, they won't have that auth scope, even if they try to authenticate.